examples/deploy-k8s/README.md

```diff
@@ -0,0 +1,60 @@
+# Deploy Ollama to Kubernetes
+
+## Prerequisites
+
+- Ollama: https://ollama.ai/download
+- Kubernetes cluster. This example will use Google Kubernetes Engine.
+
+## Steps
+
+1. Create the Ollama namespace, daemon set, and service
+
+    ```
+    kubectl apply -f cpu.yaml
+    ```
+
+1. Port forward the Ollama service to connect and use it locally
+
+    ```
+    kubectl -n ollama port-forward service/ollama 11434:80
+    ```
+
+1. Pull and run `orca-mini:3b`
+
+    ```bash
+    OLLAMA_HOST=https://<name>.fly.dev ollama run orca-mini:3b
```

Comment:
Is `fly.dev` a typo here? Seems like it should be the cluster IP.

Reply:
Yes you're completely right. This was c+p from the fly example